Get document calls flask

alyssatownsend opened the pull request:
- successfully calls flask
- just sends static JSON string
- gets back 200 msg & a Colored Page Object JSON string
- the show method returns said JSON string
- hasColorProfileTest in ProofOfTestingTest.php errors

Commit:
added dummy info

- added dummy info to send to flask to color
- created test stub

Commit:
documents.show calls flask

- call to flask in DocumentController
- route in api.php
- test in test folder

Commit:
deleted file to facilitate merge

- deleted DocumentController.php so i can merge changes from main onto this branch

Commit:
Merge branch 'main' into get-document-calls-flask

Commit:
added show function back into merged DocumentController

Commit:
added imports for DocumentController

- added imports
- added getText fcn for Document

Commit:
successfully sent get request to flask!

- getColoredPageObjectTest.php assertion passed
- flask debug terminal showed response of 200 OK

elijahbigham reviewed (changes requested):
in the show function, it should be $text = $this->uploadedDocument->getText()
For our document we want to reference a single, static document

Commit:
show sends dynamic info to flask

- instead of taking in Document, takes in text string & color profile string
- concatenates them and sends as JSON string in request body

elijahbigham reviewed (commented):
Where / how will the store method get a reference to the color profile?

Commit:
Merge branch 'main' into get-document-calls-flask

Commit:
documents.show takes in user & document id

- code for getting text from a specific document commented out b/c that doesn't work yet
- gives static color profile b/c no code yet to get a color profile

Commit:
updated DocumentController.php

- uncommented code that dynamically gets document text to color
- need to use server to test

Commit:
updated DocumentController.php

- took user id out of show parameters
